autogpt_platform/frontend/src/components/NodeOutputs.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { ContentRenderer } from "./ui/render";
+import { beautifyString } from "@/lib/utils";
+import * as Separator from "@radix-ui/react-separator";
+
+type NodeOutputsProps = {
+  title?: string;
+  truncateLongData?: boolean;
+  data: { [key: string]: Array<any> };
+};
+
+export default function NodeOutputs({
+  title,
+  truncateLongData,
+  data,
+}: NodeOutputsProps) {
+  return (
+    <div className="m-4 space-y-4">
+      {title && <strong className="mt-2flex">{title}</strong>}
+      {Object.entries(data).map(([pin, dataArray]) => (
+        <div key={pin} className="">
+          <div className="flex items-center">
+            <strong className="mr-2">Pin:</strong>
+            <span>{beautifyString(pin)}</span>
+          </div>
+          <Separator.Root className="my-4 h-[1px] bg-gray-200" />
```

Comment:
Separator root here is replicated in many places with the exact same styling. I think it makes sense to create a component out of this

Reply:
Added